splinepy/bezier.py

```diff
@@ -4,6 +4,61 @@
 class BezierBase(spline.Spline):
     """Bezier Base. Contain extra operations that's only
     available for bezier families.
+
```

Comment:
Can you please add the "s" in the line above, I know it was probably my mistake, but I just noticed it :wink:

Reply:
Probably my mistake. Also "'re"  (`Contains extra operations that're --`)

Reply:
Sorry guys, I didn't spot the mistakes while extending the documentation, otherwise I would have changed it for sure, now I'm gonna adapt it after this review :)